doc/source/migrations/index.rst

```diff
@@ -21,11 +21,23 @@ Development version
 
 **Breaking changes:**
```

Comment:
Does the migration guide need to be updated for v7? & should these breaking changes be in the "development version" section or "v8" (even though it isn't released yet)

Reply:
Breaking changes should go in the dev section for now. When releasing we will update it.